pkg/kv/kvclient/kvcoord/txn_interceptor_pipeliner.go

```diff
@@ -21,10 +21,10 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/log"
 	"github.com/cockroachdb/errors"
 	"github.com/cockroachdb/redact"
-	"github.com/google/btree"
+	gbtree "github.com/google/btree"
 )
 
-// The degree of the inFlightWrites btree.
+// The degree of the inFlightWrites gbtree.
```

Comment:
[nit] I think this one could probably stay `btree`.

Reply:
Done.